libflux/go/libflux/parser_linux.go

```diff
@@ -0,0 +1,4 @@
+package libflux
+
+// #cgo LDFLAGS: -ldl
```

Comment:
Can I ask what library does `-ldl` include?

Reply:
It's `libdl` which manages `dlopen`, `dlsym`, and `dlclose`. For whatever reason, this was required on Linux and wasn't required on Mac OS X. The Rust library appeared to use it for whatever reason.